CONTRIBUTING.md

```diff
@@ -0,0 +1,150 @@
+# **Contributing Guidelines**
+## **Befor Contributing**
+
+Welcome to [Autobot!](https://github.com/OSCA-Kampala-Chapter/autobot) Before sending your pull requests, make sure that you **read the whole guidelines.** If you have any doubt on the contributing guide, please feel free to [state it clearly in an issue](https://github.com/OSCA-Kampala-Chapter/autobot/issues) or ask the community in [Discord](/docs/404.md) 
+
+## **Contributing**
+### **Contributor**
+We are very happy that you are considering implementing autobot for others! Being one of our contributors on this repository, you agree and confirm
+-   You did your work - no plagiarism allowed
+    -    Any plagiarized work will not be merged.
+-   Your work will be distributed under [MIT](/LICENSE) License once your pull request is merged
+-   Your submitted work fulfils or mostly fulfils our styles and standards
```

Comment:
seems like we have a typo here `fulfils` instead of `fulfills`

Reply:
Resolved [solution here](https://github.com/OSCA-Kampala-Chapter/autobot/pull/114/commits/436c4620f41f3964f69aa28f49e083871b687d72)